codebuild/codebuild_terraform_apply/code_build_terraform.tf

```diff
@@ -0,0 +1,58 @@
+resource "aws_codebuild_project" "code_pipeline_terraform" {
+  name        = "code-pipeline-terraform"
```

Comment:
I wonder if we should pass in the codepipeline name to these modules so that we can set the names of these to something specific. If we've got a pipeline that deploys keycloak to staging and then production with terraform we probably want these to appear in the AWS console as `keycloak-terraform-staging` and `keycloak-terraform-production`. Might be good to pass in both the pipeline name and environment as vars.

Reply:
Sounds good, I'm adding it now